docs/source/using/experimental-features.adoc

```diff
@@ -30,11 +30,14 @@ By default, {project} does not expose all link:https://github.com/openshift/orig
 
 You can set the `MINISHIFT_ENABLE_EXPERIMENTAL` environment variable to enable the following options for the xref:../command-ref/minishift_start.adoc#[`minishift start`] command:
 
-`service-catalog`::
-Enables provisioning the OpenShift link:https://docs.openshift.org/latest/architecture/service_catalog/index.html[service catalog].
-
```

Comment:
We have removed the `service-catalog` flag from Minishift . Refer https://github.com/minishift/minishift/commit/15b69a34d488d6414dad5b16869dbcfd16faaa05